app/src/App.js

```diff
@@ -78,7 +81,7 @@ const App = () => {
               <Nav className="justify-content-end" activeKey="/home">
                 <Nav.Item>
                   <Nav.Link active>
-                    <Link to="/day/:date">Day View</Link>
+                    <Link to="/day/:day">Day View</Link>
```

Comment:
Currently the link/ route of 
`/day/:day` does not work 

`Cannot GET /day/:day`
So will need to fix the route somewhere. Maybe where you change the route from data to day needs to be double checked

Reply:
Thank you! I'll look into it. 👀